.travis.yml

```diff
@@ -133,6 +135,7 @@ jobs:
           file:
            - "inspectit-oce-agent/build/inspectit-oce-agent-$RELEASE_VERSION.jar"
            - "inspectit-oce-documentation/build/distributions/inspectit-oce-documentation-$RELEASE_VERSION.zip"
+           - "inspectit-oce-demo/build/inspectit-oce-demo-$RELEASE_VERSION.zip"
```

Comment:
I think this has to be ``inspectit-oce-demo/build/distributions/inspectit-oce-demo-$RELEASE_VERSION.zip``